### PR TITLE
Create formal agreement

### PR DESCRIPTION
**Pull request by elena-vi**
## Context

Adding the front end ability to add a formal agreement 
## Things to check

- [ ] Environment variables have been updated

### CODE REVIEW

**Comment by C-gyorfi** [comment]
You need to enable both formal and informal agreements feature flag for the test to pass, its not necessary a good `design` but currently formal agreements depends on informal agreements feature flags too

File: app/views/agreements/form/_agreement_formal.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <% end %>
+  </fieldset>
+</div>
+
+
+<%= render :partial => "agreements/form/agreement_informal" %>
+
+
+
+<%#= submit_tag 'Next', class: 'button' %>
```
Comment (C-gyorfi) [inline app/views/agreements/form/_agreement_formal.html.erb:24]: This is commented out

File: spec/features/income_collection/agreements/creating_informal_agreement_spec.rb [hunk 2]
```diff
@@ -293,4 +300,23 @@ def stub_cancel_agreement_response
...
+
+  def stub_court_cases
+    stub_request(:get, 'https://example.com/income/api/v1/court_cases/1234567%2F01/')
+        .with(
+          headers: { 'X-Api-Key' => ENV['INCOME_API_KEY'] }
+        )
+        .to_return(
+          status: 200,
+          body: {
+              courtCases: [{
```
Comment (C-gyorfi) [inline spec/features/income_collection/agreements/creating_informal_agreement_spec.rb:312]: Minor thing, but I guess we don't need to stub a court case here could just respond with 
```RUBY
body: {
     courtCases: []
}
```

Comment (elena-vi) [reply spec/features/income_collection/agreements/creating_informal_agreement_spec.rb:312]: my bad

**Pull request merged**